vision_datasets/common/data_manifest.py

```diff
@@ -447,6 +452,14 @@ def create_dataset_manifest(coco_file_path_or_url, data_type, container_sas_or_r
 
         file_reader.close()
 
+        if data_type == DatasetTypes.IMCAP:
+            images_by_id = {img['id']: ImageDataManifest(img['id'], get_full_sas_or_path(img['file_name']), None, None, []) for img in coco_manifest['images']}
```

Comment:
btw, for image width and height. Those are two typical info supported by COCO, not specific to caption or classification, or sth. It is ok to allow for missing values in caption case, but can we respect and take the info, if it is provided? Thanks

Reply:
The ground truth file that Xiaowei has shared with us does not contain the image width and height info. The width and height info are missing in the gt file, unless we want to add these info to the ground truth file.

Reply:
Not just Xiaowei is working on captioning...there could be people who works with captioning that do have width and height.

Reply:
"It is ok to allow for missing values in caption case, but can we respect and take the info, **if it is provided**?"

What I mean is 
    respecting the info, if it is provided
not
   always requires width and height

Reply:
I see. Have added a logic to check whether width and height are provided:

`            for img in coco_manifest['images']:
                image_width = None if 'width' not in img else img['width']
                image_height = None if 'height' not in img else img['height']
                images_by_id[img['id']] = ImageDataManifest(img['id'], get_full_sas_or_path(img['file_name']), image_width, image_height, [])`